rnn_param_search.txt

```diff
@@ -0,0 +1,47 @@
+Wednesday 02/05
```

Comment:
Does this file need to be in the repository?

Reply:
@tblah do you know if this file needs to be in the repo?

Reply:
I'm going to accept this merge request so we can get the code in and we can always remove this later.